js/binance.js

```diff
@@ -784,8 +784,17 @@ module.exports = class binance extends Exchange {
         });
     }
 
+    costToPrecision (symbol, cost) {
```

Comment:
I made a mistake in the commit comment.
The thing that changed was in fact using precision.quote instead of precision.price, not ROUND -> TRUNCATE as commented.
I'll fix that up and push again.
And I'll check out the unexpected CI failure...